site/docs/v1/tech/developer-guide/api-gateways/aws-api-gateway.adoc

```diff
@@ -0,0 +1,217 @@
+---
+layout: doc
+title: Amazon API Gateway
+description: Amazon API Gateway OIDC integration
+navcategory: developer
+---
+
+== Overview
+
+Amazon API Gateway is an AWS service for creating, publishing, maintaining, monitoring, and securing REST, HTTP, and WebSocket APIs at any scale. You can create APIs for AWS services or other web services.
+
+You can configure Amazon API Gateway to handle authentication and authorization to a resource through JSON Web Tokens (JWTs) issued on behalf of a user by an identity provider.
+
+In this document, you'll learn how to set up Amazon API Gateway and FusionAuth as the identity provider to protect a lambda function running on your AWS cloud instance.
+
+== Prerequisites
+
+* A FusionAuth instance running on a publicly accessible URL. You can spin up a link:/pricing[basic FusionAuth Cloud instance] or link:/docs/v1/tech/installation-guide/[install it on any server].
```

Comment:
👍

Reply:
Can you add something about exposing a local FusionAuth using ngrok or a similar tool?